### ISSUE TITLE
Create Event Sequence Diagram Review

### ISSUE THREAD

**Issue by didemongu** [bounswe/bounswe2018group9#90]
I prepared sequence diagram for creating event. Here is this:
![create event](https://user-images.githubusercontent.com/36166597/37794663-30de0646-2e23-11e8-81d3-9d980417c4bf.png)
What should be changed in the diagram?

**Comment by serkan-ozel-2015400123** [comment]
@didemongu  Instead of registered user, we can use exactly same name in class diagrams which is signed user. 
@akaogluozgur There is no createEvent(eventInfo) function for signedUser, either didem or you should change the diagrams. In didem's diagram signeduser have createEvent function. I think this is better than event has create function. 
@akaogluozgur Member user also should have a function that sends location data to Google API, I suppose.
@akaogluozgur  Finally, an event should have request function for database record creation of the event.

**Comment by didemongu** [comment]
I will replace registered user with signed user 👍🏼

**Comment by nazmicancalik** [comment]
@serkan-ozel-2015400123  your observations are correct. But what you suggest to @akaogluozgur may be the methods of the manager classes. We have to reach to a consesus here: 
* Are we going to put those methods into entity type of classes
* or we create the manager classes
Sorry for the inconvenience, can someone clear me on this issue, I am a bit off track

**Comment by nazmicancalik** [comment]
@didemongu I have a question: shouldn't the event class on the sequence diagram you draw should start after creation, in other words shouldn't it start existing  from the half of the page ?

**Comment by serkan-ozel-2015400123** [comment]
And let me add, I learned from internet that vertical blocks are for messages only. Not for object lifeline. You can use dashed line. So actually this type of drawing is not correct.

**Comment by nazmicancalik** [comment]
@serkan-ozel-2015400123 You are right, dashed line represents the object lifetime  but vertical blocks represent *the time component is working* and I think some of the vertical blocks are true in Didem's case. 

Resource: [Wikipedia/SequenceDiagram](http://www.wiki-zero.com/index.php?q=aHR0cHM6Ly9lbi53aWtpcGVkaWEub3JnL3dpa2kvU2VxdWVuY2VfZGlhZ3JhbQ)
Third Pharagraph, Last Sentence

**Comment by serkan-ozel-2015400123** [comment]
Can you update please ? :) @didemongu

**Comment by didemongu** [comment]
I updated it
